.github/ISSUE_TEMPLATE/internal.yml

```diff
@@ -0,0 +1,9 @@
+name: 💡 [Internal] Blank Issue
+description: Only for Sentry Employees! Create an issue without a template.
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+    validations:
+      required: true
```

Comment:
I'd still enforce this because issues without any description are a pain, no matter who creates them